plugin/src/main/java/spp/jetbrains/sourcemarker/status/BreakpointStatusBar.java

```diff
@@ -155,13 +168,13 @@ private void removeActiveDecorations() {
         SwingUtilities.invokeLater(() -> {
             if (expandLabel != null) expandLabel.setIcon(PluginIcons.expand);
             closeLabel.setIcon(PluginIcons.close);
-            configPanel.setBackground(PluginColors.getBackgroundDefault());
+            configPanel.setBackground(spp.jetbrains.sourcemarker.PluginUI.getBackgroundDefaultColor());
```

Comment:
You have several fully qualified references in this PR. Please remove

Reply:
Fixed